app/controllers/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+    return if @user
+
+    flash[:danger] = "User not found!"
+    redirect_to root_path
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = "Welcome to the sample app!"
+      redirect_to @user
+    else
+      flash[:danger] = "Account registration failed"
```

Comment:
` flash[:danger] = "Account registration failed"` change to ` flash.now[:danger] = "Account registration failed"`

Reply:
done ms